WordPress/UITestsFoundation/Screens/Me/ContactUsScreen.swift

```diff
@@ -0,0 +1,70 @@
+import ScreenObject
+import XCTest
+
+/// This screen object is for the Support section. In the app, it's a modal we can get to from Me
+/// > Help & Support > Contact Support, or, when logged out, from Prologue > tap either continue button > Help > Contact Support.
+public class ContactUsScreen: ScreenObject {
+
+    private let closeButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["ZDKbackButton"]
+    }
+
+    private let sendButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["ZDKsendButton"]
+    }
+
+    private let attachButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["ZDKattachButton"]
+    }
+
+    private let deleteMessageButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Delete"]
+    }
+
+    var closeButton: XCUIElement { closeButtonGetter(app) }
+    var sendButton: XCUIElement { sendButtonGetter(app) }
+    var attachButton: XCUIElement { attachButtonGetter(app) }
+    var deleteMessageButton: XCUIElement { deleteMessageButtonGetter(app) }
+
+    public init(app: XCUIApplication = XCUIApplication()) throws {
+        // Notice we are not checking for the send button because it's visible but not enabled,
+        // and `ScreenObject` checks for enabled elements.
+        try super.init(
+            expectedElementGetters: [
+                closeButtonGetter,
+                attachButtonGetter,
+            ],
+            app: app
+        )
+    }
+
+    public func assertCanNotSendEmptyMessage() -> ContactUsScreen {
+        XCTAssert(!sendButton.isEnabled)
+        return self
+    }
+
+    public func assertCanSendMessage() {
```

Comment:
For the consistency, what do you think of making this return `ContactUsScreen` too?

Reply:
Sure! Done in [6f21fa2](https://github.com/wordpress-mobile/WordPress-iOS/pull/17504/commits/6f21fa27936dc20abfe3da01d44285d3e421ec05).